Resources/Prototypes/Entities/Objects/Weapons/Throwable/radpen.yml

```diff
@@ -0,0 +1,34 @@
+- type: entity
+  parent: BaseItem
+  id: RadPenBox
+  name: radioactive pen box
+  description: A small box containing a radioactive pen. Packaging disintegrates when opened, leaving no evidence behind.
+  components:
+  - type: Item
+    size: Tiny
+  - type: Sprite
+    sprite: Objects/Storage/penbox.rsi
+    state: radpen
+  - type: SpawnItemsOnUse
+    items:
+    - id: PenRadioactive
+    sound:
+      path: /Audio/Effects/unwrap.ogg
+
+- type: entity
+  name: pen
+  suffix: Radioactive
+  parent: PenEmbeddable
+  description: A dark ink pen.
+  id: PenRadioactive
+  components:
+  - type: ItemToggle
+    soundActivate:
+      path: /Audio/Items/flashlight_on.ogg
+    soundDeactivate:
+      path: /Audio/Items/flashlight_off.ogg
```

Comment:
Maybe give it other sounds that are a little better to distinguish for turning on and off. I toggled it a bunch of times and could no longer tell if it was on or off afterwards. But you could call it a feature I guess.

Reply:
I like the green glow idea since it does the job to solve a lot of issues without ruining the item. I'll add that on and see how it looks.

Reply:
I could potentially make it warm to the touch as well